sdk/anomalydetector/azure-ai-anomalydetector/CHANGELOG.md

```diff
@@ -1,14 +1,27 @@
 # Release History
 
-## 3.0.0-beta.5 (Unreleased)
+## 3.0.0-beta.5 (2022-12-01)
 
 ### Features Added
 
+- Added `OneTable` and `MultiTable` two data schemas.
+- Added `topContributorCoun` in the request body for `getMultivariateBatchDetectionResult`, `detectMultivariateBatchAnomaly` and `detectMultivariateLastAnomaly`.
+
 ### Breaking Changes
 
-### Bugs Fixed
+- Renamed `trainMultivariateModelWithResponse` to `createAndTrainMultivariateModel`
```

Comment:
remember to update to latest name, e.g. currently it is `trainMultivariateModel`

You can do that in another PR.

Reply:
updated.

Reply:
The PR is already merged. Please create another PR target main branch.